cinema/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=255, null=True, blank=True)
+    duration = models.IntegerField()
+
+    class Meta:
+        verbose_name = "movie"
+        verbose_name_plural = "movies"
```

Comment:
I'm not sure you really need it, because django do it well by itself, but, ok, why not